### PR TITLE
WIP: [NEW] add carto entry for web and mobile app 

### PR DESCRIPTION
**Pull request by njeudy**
First release of carto implemetation

TODO: a way to have dynamic map url.

### CODE REVIEW

File: src/components/Nav.vue [hunk 1]
```diff
@@ -31,13 +35,18 @@
...
-
-        <div v-if="getLog && hasUserAccountValidationRights" class="navbar-item has-dropdown is-hoverable">
-          <a class="navbar-link">
-            Admin
-          </a>
+        <router-link to="/carto" class="navbar-item">
+          <a class="navbar-link"> Carto </a>
+        </router-link>
+        <router-link to="/" class="navbar-item" v-if="!getLog">
+          <a class="navbar-link"> Se connecter </a>
```
Comment (seegwen) [inline src/components/Nav.vue:43]: There, removing the extra `<a class="navbar-link"`

**Review by seegwen — CHANGES_REQUESTED** [review]
There are a lot of indentation fixes. Though I tend to do the same and fix what's under my keyboard, I think what Valentin told me is a good remark though and I try to follow the advice: To try and keep one and only one concern for each PR. 

Betterwise, I think the lint PR #63 will do what you just did, but on the entire codebase.

I don't mind for this PR.

Can you address my comments about the links in the Nav, and then I'll validate and we can merge.

File: src/components/Nav.vue [hunk 2]
```diff
@@ -31,13 +35,18 @@
...
     >
-
       <div class="navbar-end">
-
-        <div v-if="getLog && hasUserAccountValidationRights" class="navbar-item has-dropdown is-hoverable">
-          <a class="navbar-link">
-            Admin
-          </a>
+        <router-link to="/carto" class="navbar-item">
+          <a class="navbar-link"> Carto </a>
```
[the remark this answers is not in the document]
Comment (njeudy) [reply src/components/Nav.vue:40]: ok

**Comment by njeudy** [comment]
@seegwen Ok code updated :)

**Review by seegwen — APPROVED** [review]

**Pull request merged**